From adc4b51aa913991a44cdd4242c6053dfc49f6d36 Mon Sep 17 00:00:00 2001
From: Peter Harliman Liem <peter.harliman.liem@intel.com>
Date: Thu, 28 Jun 2018 15:59:16 +0800
Subject: [PATCH] DRVLIB_SW-801 - Fix bug in ltq_system_reset compatible string

Compatible strings was incorrect, they should be placed in
separate array index.
---
 drivers/char/ltq_reset/ltq_system_reset.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/char/ltq_reset/ltq_system_reset.c b/drivers/char/ltq_reset/ltq_system_reset.c
index 07e6ee675cab..fd0065d5d0d3 100644
--- a/drivers/char/ltq_reset/ltq_system_reset.c
+++ b/drivers/char/ltq_reset/ltq_system_reset.c
@@ -2466,7 +2466,8 @@ static int sysrst_remove(struct platform_device *pdev)
 }
 
 static const struct of_device_id sysrst_match[] = {
-	{ .compatible = "intel,sysrst-xrx500", "intel,sysrst-falconmx" },
+	{ .compatible = "intel,sysrst-xrx500" },
+	{ .compatible = "intel,sysrst-falconmx" },
 	{},
 };
 MODULE_DEVICE_TABLE(of, sysrst_match);
